specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -118,28 +98,17 @@ function TreeView<SCHEMA extends AnyTree>({
     [baseUrl, sortField]
   );
 
-  const statsThreshold = getPref(
-    `TreeEditor.Rank.Threshold.${tableName as 'Geography'}`
-  );
-  const getStats = React.useCallback(
-    async (nodeId: number | 'null', rankId: number): Promise<Stats> =>
-      rankId >= statsThreshold
-        ? fetchStats(`${baseUrl}/${nodeId}/stats/`)
-        : Promise.resolve({}),
-    [baseUrl, statsThreshold]
-  );
-
   const [rows, setRows] = useAsyncState<RA<Row>>(
     React.useCallback(async () => getRows('null'), [getRows]),
     true
   );
-  const id = useId('tree-view');
 
   // FEATURE: synchronize focus path with the URL
   const [focusPath = [], setFocusPath] = useCachedState(
```

Comment:
the focus paths should be independent for both trees
as far as caching, store the most recently focused tree's focused path in this cache

<details>
<summary>Solution</summary>
two react states, and a useEffect that updates local storage on change to focused tree's focus path
</details>